style.css

```diff
@@ -1109,20 +1109,23 @@ button.toggle {
 /* COLOR */
 
 .color-accent,
+.color-accent-hover:focus,
```

Comment:
You need to update the targeting for ['content']['accent'] in`twentytwenty_get_elements_array()` to also include the new helper class targets: `.color-accent-hover:focus`, `.bg-accent-hover:focus` and `.border-color-accent-hover:focus`.

Reply:
Thanks, done in c2c64697def9fd8cc0c42f2bc34d408d99ceaabd